Tests/FluentAssertions.Json.Specs/JTokenFormatterSpecs.cs

```diff
@@ -7,62 +7,40 @@ namespace FluentAssertions.Json.Specs
     // ReSharper disable InconsistentNaming
     public class JTokenFormatterSpecs
     {
+        public JTokenFormatterSpecs()
+        {
+            Formatter.AddFormatter(new JTokenFormatter());
```

Comment:
Just FYI, this might make tests dependent on execution order.

Reply:
In this case, all tests in this class need it. And any test that relies on `JTokenAssertions` will get it [by default](https://github.com/fluentassertions/fluentassertions.json/blob/17997feb5c466301d32e4bca94a4b03756422cb5/Src/FluentAssertions.Json/JTokenAssertions.cs#L23).